src/frontend/templates/cart.html

```diff
@@ -18,43 +18,45 @@ <h3>{{ len $.items }} item
-                                <a class="btn btn-info" href="/" role="button">Browse more products &rarr; </a>
+                                <button class="btn btn-secondary empty-btn" type="submit">Empty cart</button>
+                                <a class="btn btn-info" href="/" role="button">Keep browsing</a>
                             </form>
                     
                         </div>
                     </div>
                     <hr>
                     
                     {{ range $.items }}
-                    <div class="row pt-2 mb-2">
-                        <div class="col text-right">
-                                <a href="/product/{{.Item.Id}}"><img class="img-fluid" style="width: auto; max-height: 60px;"
-                                    src="{{.Item.Picture}}" /></a>
-                        </div>
-                        <div class="col align-middle">
-                            <strong>{{.Item.Name}}</strong><br/>
-                            <small class="text-muted">SKU: #{{.Item.Id}}</small>
-                        </div>
-                        <div class="col text-left">
-                            Qty: {{.Quantity}}<br/>
-                            <strong>
-                                {{ renderMoney .Price}}
-                            </strong>
+                    <div class="product-item">
+                        <div class="row pt-2 mb-2">
+                            <div class="col text-right image">
+                                    <a href="/product/{{.Item.Id}}"><img class="img-fluid"
+                                        src="{{.Item.Picture}}" /></a>
+                            </div>
+                            <div class="col text-left text">
+                                <h4>{{.Item.Name}}</h4>
+                                <p><small class="text-muted">SKU: #{{.Item.Id}}</small></p>
+                                <div class="details">
+                                    Quantity: {{.Quantity}}<br/>
+                                    <strong>
+                                        USD {{ renderMoney .Price}}
+                                    </strong>
+                                </div>
+                            </div>
                         </div>
                     </div>
                     {{ end }} <!-- range $.items-->
                     <div class="row pt-2 my-3">
-                        <div class="col text-center">
-                            <p class="text-muted my-0">Shipping Cost: <strong>{{ renderMoney .shipping_cost }}</strong></p>
-                            Total Cost: <strong>{{ renderMoney .total_cost }}</strong>
+                        <div class="col text-center order-summary">
+                            <p class="text-muted my-0">Shipping Cost: <strong>USD {{ renderMoney .shipping_cost }}</strong></p>
+                            Total Cost: <strong>USD {{ renderMoney .total_cost }}</strong>
```

Comment:
This part is a little odd. Don't we support currencies? If so, why is this version hardcoding USD to the cart UI?

Reply:
Fixed